src/business.ts

```diff
@@ -168,4 +173,36 @@ export class BusinessApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take a Business token id, some Business attributes, and update
+   * the Business with that Id in Marqeta with these values. The return value will be
+   * the updated Business.
+   */
+  async update(businessTokenId: string, business: Partial<Business>): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
+    const updateOptions = cleanJson(
+      snakecaseKeys(business)
+    )
+    const resp = await this.client.fire('PUT',
+      `businesses/${businessTokenId}`,
+      {},
+      { ...updateOptions }
```

Comment:
At this point, it's easy enough to say:

```typescript
     const resp = await this.client.fire('PUT',
       `businesses/${businessTokenId}`,
       undefined,
       updateOptions,
     )
```

as we don't need to make another copy of this... and the query params is easily ignored if it's passed in as `undefined`.